### PR TITLE
fix: use specific git version and setup git for container use

### LINKED ISSUE #26: v2.x fails with missing git version
**Describe the bug**
Our builds using this action started failing with the following message:

```
Step 2/4 : RUN apt-get update && apt-get install --no-install-recommends -y git=1:2.30.2-1 && rm -rf /var/lib/apt/lists/*
   ---> Running in a471eb992ff5
  Get:1 http://deb.debian.org/debian bullseye InRelease [116 kB]
  Get:2 http://deb.debian.org/debian-security bullseye-security InRelease [[48](https://github.com/TBCTSystems/bct-common-licensing-softwarekey/actions/runs/4852448319/jobs/8647443878#step:2:48).4 kB]
  Get:3 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
  Get:4 http://deb.debian.org/debian bullseye/main amd64 Packages [8183 kB]
  Get:5 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [237 kB]
  Get:6 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.6 kB]
  Fetched 8643 kB in 1s (6962 kB/s)
  Reading package lists...
  Reading package lists...
  Building dependency tree...
  Reading state information...
  E: Version '1:2.30.2-1' for 'git' was not found
  The command '/bin/sh -c apt-get update && apt-get install --no-install-recommends -y git=1:2.30.2-1 && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
```

**To Reproduce**

Use the action, our code as follows, but the error occurs during building the image from the Dockerfile:

```yaml
- name: 🖼️ Export drawio files to png files
      uses: rlespinasse/drawio-export-action@v2.x
      with:
        format: png
        transparent: true
        embed-diagram: true
        path: ./docs/drawio/
        output: ./images
        action-mode: all
```

**Expected behavior**
Image builds successfully when using action

### PR DESCRIPTION
**Pull request by rlespinasse**
Resolves #26

### CODE REVIEW

**Review by rlespinasse — COMMENTED** [review]
n.a

**Review by mortkarlberg — DISMISSED** [review]

**Review by rlespinasse — COMMENTED** [review]
n.a

File: drawio-export.sh [hunk 1]
```diff
@@ -50,8 +50,11 @@ fi
 
 # Need of full clone execpt for 'all' action mode
 if [ "${INPUT_ACTION_MODE}" != "all" ]; then
+  # To correctly configure git setup inside a container
+  # See https://github.com/actions/checkout/issues/1169
+  git config --system --add safe.directory '/github/workspace'
   # Need a classic clone of the repository to work with
-  # but 'actions/checkout' make a shallow clone by default
+  # but 'actions/checkout' make a shallow clone by default$
```
Comment (specialpointcentral) [inline drawio-export.sh]: `$` maybe a typo?

Comment (rlespinasse) [inline drawio-export.sh]:
```suggestion
  # but 'actions/checkout' make a shallow clone by default
```

**Pull request merged**